frontend/src/entry-forms/Transaction.svelte

```diff
@@ -41,34 +42,14 @@
     }
   }
 
-  /// Extract tags and links that can be provided in the narration <input>.
-  function onNarrationChange({
-    currentTarget,
-  }: {
-    currentTarget: HTMLInputElement;
-  }) {
-    const { value } = currentTarget;
-    entry.tags = [...value.matchAll(TAGS_RE)].map((a) => a[1] ?? "");
-    entry.links = [...value.matchAll(LINKS_RE)].map((a) => a[1] ?? "");
-    entry.narration = value
```

Comment:
this needs to be re-added. we need to remove the extracted tags and links here

Reply:
I added it again, sorry that you had to check it when the actual changes I wanted to share weren't present.